assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml

```diff
@@ -0,0 +1,21 @@
+- name: example ec2 group2
+  ec2_group1:
```

Comment:
I couldn't find a module named `ec2_group1`. Perhaps you meant `ec2_group`

Reply:
I changed it for testing. I was having an issue, with document results.